modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/data_set.jsp

```diff
@@ -22,11 +22,11 @@ renderResponse.setTitle(ParamUtil.getString(request, "dataSetLabel"));
 		HashMapBuilder.<String, Object>put(
 			"backURL", backURL
 		).put(
-			"dataSetERC", ParamUtil.getString(request, "dataSetERC")
+			"cellClientExtensionRenderers", fdsAdminDisplayContext.getFDSCellRendererCETsJSONArray()
```

Comment:
There's something odd in the naming with `cellClientExtensionRenderers` and `filterClientExtensionRenderers`. I understand we want to get rid of FDS prefix in entity naming, however, references to FDS CX should not be renamed because the CX themselves are created for/belong to/do apply to FDS as component. 

DSM just makes use of them to prepare proper configuration. Nothing changed in the essence of these CX because DSM decided to rename the persistence model.

If, for consistency reasons, we want to not include any FDS-naming thing in the persistence, we shall at least change naming a bit: 

- `cellRendererClientExtensions` , because it renders a cell, and 
- `filterClientExtensions`, because it implements a filter, not just its rendering but also the odata preparation

Above look more solid names. This entails renaming of the interface `IClientExtensionRenderer` to just `IClientExtension` but I believe it's ok

Reply:
@dsanz  Thinking about above, couple of additional points:
- It might we worth considering completely abandoning `*Renderer` naming pattern, replacing it with `ClientExtension`. We could have something like `ExternalClientExtensions` and `InternalClientExtensions`. `Renderer` has proved to be quite unintuitive. This would be quite a large refactor, I'd say beyond the scope of this task.
- This rename is not related to DSM persistence, so we can easily tweak it later on. In persistence, the fields that relate to cells are `Renderer` and `RendererType` fields in  `DataSetTableSection`. In the long run, this might not be enough, as there could be multiple client extensions / renderers in the scope of one table section. This somewhat combines with my first point.

Reply:
> It might we worth considering completely abandoning *Renderer naming pattern, replacing it with ClientExtension.

I agree, the amount and nature of the FDS CXs (both present and future) makes me believe that using just CX or FDS*CX is more than fine

My point was about naming specific cx types inside the DSM, but as you said, we have a big variety so we can remove the "renderer" wording in favor of specific cx names under a general interface just called ClientExtensions or the like.

As this PR is much more important than this naming thing, which we can decide about later on, I'm ok with pushing as it is